zerver/webhooks/wordpress/doc.md

```diff
@@ -1,33 +1,52 @@
+# Zulip WordPress integration
+
 Get WordPress notifications in Zulip!
 
-If you're hosting your WordPress blog yourself (i.e., not on WordPress.com),
-first install the
-[HookPress plugin](https://wordpress.org/plugins/hookpress/) (experimental).
+!!! tip ""
+
+    If you're hosting your WordPress blog yourself (i.e., not on WordPress.com),
+    first install the [HookPress plugin][1] (experimental).
+
+{start_tabs}
 
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
-    **Important:** The HookPress plugin requires URL parameters to
-    be delimited by semicolons instead of ampersands. If you have a
-    self-hosted blog, replace every `&` with `;` in the URL above.
+    !!! warn ""
+
+        The HookPress plugin requires URL parameters to
```

Comment:
Removed the **Important:** prefix as it felt like it was dominating the section a bit. It is a subjective opinion, so the options to re-include it or to use the prefix "**Note:**" or to convert it to a tip may all be considered.

Reply:
Same note as above since the plugin is no longer available.